aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/states/DeleteUserState.kt

```diff
@@ -54,14 +53,21 @@ internal sealed class DeleteUserState : State {
                 }
                 is DeletingUser -> {
                     when (deleteUserEvent) {
-                        is DeleteUserEvent.EventType.SignOutDeletedUser -> StateResolution(UserDeleted())
+                        is DeleteUserEvent.EventType.UserDeleted -> {
+                            val action = deleteUserActions.initiateSignOut()
+                            StateResolution(UserDeleted(), listOf(action))
+                        }
                         is DeleteUserEvent.EventType.ThrowError -> {
-                            StateResolution(Error(deleteUserEvent.exception))
+                            if (deleteUserEvent.signOutUser) {
```

Comment:
I don't see signOutUser property in the deleteUserEvent. Did you add in this PR?

Reply:
There is no sign out process in Delete User anymore. It proceeds from DeletingUser state to SigningOut state.